datamade/la-metro-councilmatic
# Adding functionaility to refresh Subject-GUID relationship

## Overview

This PR:

- Adds in a new model `SubjectGuid` to act as a lookup model between `Subject` names and Semaphore GUIDs.
- Creates a `refresh_guid` management command that clears out all unused Subject/GUID pairs, then repopulates with current topic list.
- Creates an endpoint to run the `refresh_guid` command.

### Checklist



## Testing Instructions

 * Migrate the database.
 * Navigate to the `http://127.0.0.1:8000/refresh-guid/` endpoint. Confirm it is a 403 response.
 * Copy the `REFRESH_KEY` from `settings_deployment.py.example` into your local `settings_deployment.py`.
 * Access the endpoint now, passing in the `REFRESH_KEY` as a parameter. Confirm you receive a 200 response.

- In the terminal, run `python manage.py refresh_guid`
- Confirm items are deleted and updated via a quick exploration in the django shell.

Handles #436
